packages/at_client_mobile/pubspec.yaml

```diff
@@ -11,16 +11,16 @@ dependencies:
   path: ^1.8.2
   flutter:
     sdk: flutter
-  package_info_plus: ^4.0.2
+  package_info_plus: ^5.0.0
```

Comment:
Have you verified that this new major version of package_info_plus doesn't cause any problems?

Reply:
My bad, I had package_info_plus pinned to 4.2.0 in my app's pubspec, Hence flutter pub get was successful previously. Flutter pub get fails with the apps.

Because every version of at_client_mobile from path depends on package_info_plus ^5.0.0 and new_version >=0.1.0 depends on
  package_info_plus ^1.0.0, at_client_mobile from path is incompatible with new_version >=0.1.0.
So, because at_wavi_app depends on both new_version ^0.3.0 and at_client_mobile from path, version solving failed.

Reply:
I updated the package as the pubscore was decreased by 10 points for not updating to the latest package. Shall I revert it back ?

Reply:
Seems like we would have that problem anyway with ^4.0.2 as it was?

I think we should test the apps with package_info_plus ^5.0.0 and move this package to ^5.0.0 once we've confirmed that it doesn't cause actual issues with the apps. The apps can then all upgrade their package_info_plus dependency to ^5.0.0 also; @sachins-geekyants @Sardar1208 does that sound OK?

Reply:
SGTM

Reply:
I verified with atMospherePro and atBuzz pointing to package_info_plus: ^5.0.0. Linking the branches below
[atBuzz](https://github.com/atsign-company/atBuzz_2.0/tree/verify_atclient_mobile_updates)
[atMospherePro](https://github.com/atsign-foundation/atmosphere_pro/tree/verify_atclient_mobile_update)

And with the Wavi app there were issues while trying to build the app.
![Screenshot from 2024-02-08 11-34-42](https://github.com/atsign-foundation/at_client_sdk/assets/43639593/09ce3610-db95-4ad5-8a09-ec4030e854ba)

@sachins-geekyants can you please verify and confirm?

Reply:
@sachins-geekyants Any update on the above request from @purnimavenkatasubbu ?

Reply:
@purnimavenkatasubbu @gkc checked Buzz with `package_info_plus: ^5.0.0 `, it's working for our use case.
it will work on other atApps also.

Reply:
Thanks @sachins-geekyants. Please review the PR